core/package.json

```diff
@@ -50,7 +50,7 @@
     "lodash.debounce": "^4.0.8",
     "lru-cache": "^11.1.0",
     "lucide-react": "^0.474.0",
-    "next": "15.4.2-canary.10",
+    "next": "15.5.1-canary.4",
```

Comment:
Was upgrading Next.js required to leverage the new Runtime Cache?

Reply:
No, this was initially desirable when I thought we would move to Node.js middleware as part of this change, so I wanted to be on a version where that feature was considered "stable". Upon review I probably should have decoupled the version bump from this PR once it was clear it was not necessary.